src/scss/theme.scss

```
@@ -490,3 +491,16 @@ div {
 #small-dropdown, #small-dropdown option {
   width: 90%;
 }
+
+
+/* Paragraphs */
+p {
+  margin-bottom: 10px !important;
+}
+
+/*Highlight sections*/
+div.highlight{
+  pre {
+    margin: 10px 0 30px 0;
+  }
+}
```

```suggestion
}

```

Missing a newline at the end of this file